helix-core/src/test/java/org/apache/helix/controller/stages/TestRebalancePipeline.java

```diff
@@ -297,8 +297,10 @@ public void testMsgTriggeredRebalance() throws Exception {
   @Test
   public void testChangeIdealStateWithPendingMsg() throws Exception {
     String clusterName = "CLUSTER_" + _className + "_pending";
-    System.out.println("START " + clusterName + " at " + new Date(System.currentTimeMillis()));
+    HelixAdmin admin = new ZKHelixAdmin(_gZkClient);
 
+    System.out.println("START " + clusterName + " at " + new Date(System.currentTimeMillis()));
+    admin.addCluster(clusterName);
```

Comment:
What difference does this make / what does it fix?

Reply:
The new check added in `dropResource` failed the test `testChangeIdealStateWithPendingMsg` because the `clusterName` is not setup, so it breaks when it tries to drop resource. The previous test was not setup properly. 
Calling `admin.addCluster(clusterName)` beforehand did the fix.